### PR TITLE
Alignment scores after alignment

### PR DESCRIPTION
**Pull request by robinschmid**
Added new columns only to the rows.

image

### CODE REVIEW

**Comment by robinschmid** [comment]
Needs to be added to the image aligner

File: src/main/java/io/github/mzmine/datamodel/features/types/AlignExtraFeaturesType.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.github.mzmine.datamodel.features.types;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/types/AlignExtraFeaturesType.java:26]: i feel like we should put the alignment score types in a new package sicne they are currently only used there

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java [hunk 2]
```diff
@@ -164,13 +171,50 @@ public String getTaskDescription() {
     return "Join aligner, " + featureListName + " (" + featureLists.size() + " feature lists)";
   }
 
+  private static int getCandidates(final Range<Double> mzRange, final Range<Float> rtRange,
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java:174]: add javadoc that the originals list needs to be sorted

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java [hunk 3]
```diff
@@ -276,6 +325,82 @@ public void run() {
 
   }
 
+  /**
+   *
+   */
+  private void addAlignmentScores() {
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java:331]: refactor to utility or static method so we can use it in the lc image aligner too

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java (hunk 2 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java:174]: I push this to FeatureListUtils in a different PR. Might be cleaner to merge it this way and then open a separate PR

File: src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java (hunk 3 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/dataprocessing/align_join/JoinAlignerTask.java:331]: Done in #1021

**Comment by robinschmid** [comment]
All done in #1021
Because the clean up and utility methods was more complex than thought.
We should merge this and then see that the other one works

**Pull request merged**